src/js/custom_shields.js

```diff
@@ -68,6 +68,6 @@ export function bransonRoute(ctx, params) {
 }
 
 export function loadCustomShields() {
-  ShieldDraw.registerDrawFunction("branson", bransonRoute);
-  ShieldDraw.registerDrawFunction("paBelt", paBelt);
+  ShieldDraw.registerDrawFunction("branson", bransonRoute, 20 * ShieldDraw.PXR);
+  ShieldDraw.registerDrawFunction("paBelt", paBelt, 20 * ShieldDraw.PXR);
```

Comment:
`ShieldDraw.computeWidth()` should be responsible for multiplying by the pixel density, like it does for the other shapes.

Reply:
Fixed in de26f6a